frontend/src/components/FundingContent.tsx

```diff
@@ -9,30 +9,29 @@ const FundingContent = () => {
   const { t } = useTranslation("common", { keyPrefix: "Index" });
 
   return (
-    <div className="bg-base-lightest desktop:padding-y-4 tablet:padding-y-2 padding-y-1">
+    <div className="bg-base-lightest padding-y-1 tablet:padding-y-3 desktop-lg:padding-y-6">
```

Comment:
You'll note I've swapped the order of a lot of the responsive classes. Since the design system is mobile-first, it makes more sense to list the default classes first, then the smallest breakpoint followed by the classes that modify larger breakpoints. This is just a personal preference and does nothing functionally, so feel free to disagree.